### ISSUE TITLE
adopt chat list in the turbo window for the screen reader users

### ISSUE THREAD

**Issue by ilyapashuk** [jeffdapaz/VisualChatGPTStudio#111]
steps to reproduce:

1) activate a NVDA screen reader
2) open the VisualChatGptStudeo turbo window.
3) Navigate to the chat list
4) try to navigate through the chat list using arrow keys

expected result:
chat names should be spoken

actual result:
"JeffPires.VisualChatGPTStudio.ToolWindows.Turbo.ucChatItem" string is always spoken

cause:

it theams that you are using ListBox for the chat list, and you are using DataBinding and DataTemplate.

It theams that the chat name is displayed in the TextBlock inside that data template, together with the edit and delete buttons.

problem: screen reader does not know what control to read in the template.

simplest solution:

In the "ToString" method of the "JeffPires.VisualChatGPTStudio.ToolWindows.Turbo.ucChatItem" class return the chat name.

explanation:

Return value of the ToString method of the data object is automaticly sent to the accessibility systems as an object description.

Additionally, the following key bindings should be added:
1) delete, to delete the chat
2) f2 to edit the chat name.

That key bindings should be mentioned in the AutomationProperties of the corresponding buttons (for example in the HelpText). Screen reader user will be able to find them using "Object Navigation" feature of the screen reader.

Additional recomendations:

To fully adapt the window for the keyboard-only navigation, the following should be done:

1) add a hotkey to close the current tab (ctrl+w for example)
2) add a hotkey to switch (move the keyboard focus) to the tabs list

that key bindings also should be mentioned in the AutomationProperties of the corresponding controls.

**Comment by ilyapashuk** [comment]
And an other useful key binding: hotkey to move focus to the new message window of the current chat (to type the new message).

**Comment by jeffdapaz** [comment]
Ok, so, here the points:

1. Fix screen reader.

2. Add `Delete` key shortcut to delete the chat.

3. Add `F2` key shortcut to edit the chat name.

4. Include descriptions for the `Delete` and `F2` keys in the `AutomationProperties.HelpText` of the respective buttons.

5. Add `Ctrl+W` shortcut to close the current tab.

6. Add a shortcut to move focus to the tab list.

7. Include the `Ctrl+W` and tab focus shortcuts in the `AutomationProperties` of the respective controls.

8. Add a shortcut to move focus to the new message input of the current chat.

About the first item, I will see what I can do, because I don't have experience with this kind of "functionality".

About the shortcuts, I need to check the availability of each one, because maybe some shortcuts is already be used by the the Visual Studio.

And, when I get some free time, I will work on this.
